### PR TITLE
Better error handling

### PR DESCRIPTION
**Pull request by kurund**
## Summary

* Show the errors nicely with more information.

## Test Plan

* Included in the PR

### CODE REVIEW

File: src/graphql/queries/Notification.ts [hunk 1]
```diff
@@ -5,3 +5,9 @@ export const NOTIFICATION = gql`
     message @client
   }
 `;
+
+export const ERROR_MESSAGE = gql`
+  {
+    errorMessage @client
```
Comment (rathorevaibhav) [inline src/graphql/queries/Notification.ts:11]: cool 👍

File: src/config/apolloclient.ts [hunk 2]
```diff
@@ -15,12 +16,23 @@ const gqlClient = (auth_token: string | null) => {
     };
   });
 
+  const errorLink = onError(({ graphQLErrors, networkError }) => {
```
Comment (rathorevaibhav) [inline src/config/apolloclient.ts:19]: is this for developer debugging only? I see only errors getting printed in browser console.

**Comment by rathorevaibhav** [comment]
@kurund overall looking good. just one question.

File: src/config/apolloclient.ts (hunk 2 above)
Comment (kurund) [reply src/config/apolloclient.ts:19]: Yes, I have kept it for now.

**Pull request merged**